lib/vfscore/pipe.c

```diff
@@ -211,6 +216,7 @@ static unsigned long pipe_buf_read(struct pipe_buf *pipe_buf,
 
 out:
 	return to_read;
+
```

Comment:
Can you avoid introducing this blank line from the commit? I guess it is here by mistake.

Reply:
Yes, It was there by mistake. I removed it. 

Thank you for your comments!